backend/s3/s3.go

```diff
@@ -1429,6 +1444,9 @@ func (o *Object) Update(in io.Reader, src fs.ObjectInfo, options ...fs.OpenOptio
 	if o.fs.opt.ServerSideEncryption != "" {
 		req.ServerSideEncryption = &o.fs.opt.ServerSideEncryption
 	}
+	if o.fs.opt.SSEKMSKeyId != "" {
+		req.SSEKMSKeyId = &o.fs.opt.SSEKMSKeyId
```

Comment:
You need to leave `req.SSEKMSKeyId` alone and the linter shouldn't complain about it since we didn't implement it.